source/ID/concordium-did.rst

```diff
@@ -0,0 +1,741 @@
+                "conditionThreshold": [
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-1-M",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR",
+                    "publicKeyMultibase": "fVV"
+                  },
+                  "...",
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-N_M-M",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR",
+                    "publicKeyMultibase": "fZZ"
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ],
+    "authentication": [
+      "#acc-1"
+    ]
+  }
+
+.. note::
+  The ``publicKeyMultibase`` field contains a public key prefixed with ``f`` that denotes the base16 encoding.
+  See `The Multibase Encoding Scheme`_.
+
+
+Concordium Credential DID
+-------------------------
+
+The goal of the Account DID Document is to provide information about Concordium credentials, including a possibility to reference particular pieces of data, such as public keys.
+In order to do that, it specifies a `DID verification method <did-vefication-method_>`_ that reflects the credential authentication data.
+
+The Concordium Credential DID Document MUST contain the following data:
+
+- ``@context`` - the attribute that expresses context information.
+- ``id`` - the DID of the credential.
+- ``verificationMethod`` - the credential's verification method.
+- ``authentication`` - authentication method for the credential.
+
+The document MAY include any other public data of a Concordium credential.
+
+The following document defines a Concordium credential with ID ``CRED``.
+The credential has ``N`` keys and uses a threshold signature scheme requiring ``T`` signatures.
+
+.. code-block:: json
+
+  {
+    "@context": [
+      "https://www.w3.org/ns/did/v1",
+      "Concordium DID URI"
+    ],
+    "id": "did:ccd:NET:cred:CRED#credential-1",
+    "verificationMethod": [
+      {
+        "type": "VerifiableCondition2021",
+        "threshold": "T",
+        "conditionThreshold": [
+          {
+            "id": "did:ccd:NET:cred:CRED#key-1",
+            "type": "Ed25519VerificationKey2020",
+            "controller": "did:ccd:NET:cred:CRED",
+            "publicKeyMultibase": "fXX"
+          },
+          "...",
+          {
+            "id": "did:ccd:NET:cred:CRED#key-N",
+            "type": "Ed25519VerificationKey2020",
+            "controller": "did:ccd:NET:cred:CRED",
+            "publicKeyMultibase": "fYY"
+          }
+        ]
+      }
+    ],
+    "authentication": [
+      "#credential-1"
+    ]
+  }
+
+
+Smart Contract Instance DID
+---------------------------
+
+The goal of the Smart Contract Instance DID is to provide meta-data about the contract instance.
+At the moment, the main piece of meta-data is the Concordium account that send the initialization transaction.
```

Comment:
Probably the more important part is the list of entrypoints.

Reply:
Fixed.